pp/sidebarsPp.js

```diff
@@ -2689,4 +2689,4 @@ module.exports = {
       ]
     }
   ]
-};
+};
```

Comment:
what is this change?

Reply:
i dont understand why for that change. Because there is no change